README.md

```diff
@@ -231,6 +231,42 @@ On all following examples we will assume a default user has been set in the conf
 We will also assume you are using `(:use starkbank.core)` on your code ns calls to
 expose the starkbank SDK namespaces.
 
+### 5. Resource listing and manual pagination
+
+Almost all SDK resources provide a `query` and a `page` function.
+
+- The `query` function provides a straight forward way retrieve results that match the filters you inform, seamlessly retrieving all the filtered elements from the API.
```

Comment:
Agora faltou o "If you are not worried about data volume or processing time, this is the way to go."